libs/island-ui/core/src/lib/Text/Text.tsx

```diff
@@ -18,6 +18,7 @@ import {
   whiteSpace as whiteSpaceStyle,
   textAlign as textAlignStyle,
   capitalizeFirstLetter as capitalizeFirstLetterStyle,
+  disabledText,
```

Comment:
Confused here, disabled text for color changes? This is just a simple text, how can it be disabled? 🤔